apps/src/storage/firebaseStorage.js

```diff
@@ -451,18 +451,22 @@ FirebaseStorage.populateTable = function (jsonData, overwrite, onSuccess, onErro
   if (!jsonData || !jsonData.length) {
     return;
   }
-  getExistingTables(overwrite).then(existingTables => {
-    const promises = [];
-    const newTables = JSON.parse(jsonData);
-    for (const tableName in newTables) {
-      if (overwrite || (existingTables[tableName] === undefined)) {
-        const newRecords = newTables[tableName];
-        const recordsData = getRecordsData(newRecords);
-        promises.push(overwriteTableData(tableName, recordsData));
+  // Ensure rate limit counters have been initialized, so that updates to the
+  // counters/tables node will pass type definition checks in the security rules.
+  incrementRateLimitCounters()
+    .then(() => getExistingTables(overwrite))
+    .then(existingTables => {
+      const promises = [];
+      const newTables = JSON.parse(jsonData);
```

Comment:
I've made this appear as a friendly message in the applab debug console, for populateKeyValue as well as populateTable. Hopefully this will help levelbuilders if they ever edit these levels again.